src/javascripts/components/search.js

```diff
@@ -0,0 +1,94 @@
+/* global XMLHttpRequest */
+import accessibleAutocomplete from 'accessible-autocomplete'
+import lunr from 'lunr'
+
+var searchIndex = null
+var documentStore = null
+var timeout = 5 // Time to wait before giving up fetching the search index
```

Comment:
This seems like it could be quite short for a slow connection?

Reply:
up to 10 or 15, depending on the size of the index?